tests/tests/test_compressor.py

```diff
@@ -249,7 +249,7 @@ def test_cssmin(self):
         self._test_compressor('pipeline.compressors.cssmin.CSSMinCompressor',
             'css', 'pipeline/compressors/cssmin.css')
 
-    @skipUnless(settings.HAS_NODE, "requires node")
+    @skip("don't care")
```

Comment:
We have this compressor re-implemented and modified in our code base, so if it's failing, we ought to make sure that our rewrite isn't affected by the same issue.

Reply:
All of the failures that I've skipped fail in all Django versions test (1.11-2.2). I'm not aware if -- or when -- we've ever run this test suite, and as much as I agree with your concerns, I think we should add a similar test as this and the others in Ollie, so that we can catch and fix any pertinent failures.

Reply:
Or, if it's passing in travis but failing locally, I can probably help with the configuration. I actually wrote these tests; they were merged them into upstream a while back (jazzband/django-pipeline#531)

Reply:
It might just a better-documentation thing (meaning, the local tox failures), so if you have any suggestions beyond what I added to the README, I can give that a try.

Reply:
Also, as for the Java tests that I skipped, I was assuming that we didn't need any of those. If that is incorrect, I guess I can install the Java runtime, but I'd really hate to go down that rabbit hole locally.

Reply:
I believe that you need to run `npm install` from the repo root for some of these tests to pass. Beyond that, you might need to rebase jazzband/django-pipeline@3a0e3ccbabebe189c4c3dbcdbbc549e9d7cbf619 and jazzband/django-pipeline#644. If those don't do the trick, then I think it's not worth the trouble—in which case this PR LGTM

Reply:
Just saw your comment now about the java tests: we don't need these. They could perhaps be wrapped in a skipUnless for `settings.HAS_JAVA`, but that's a very nitpicky concern and I think it's also fine as is.

Reply:
I definitely did `npm install`. Unless I was too quick about it, when I stepped through this particular test failure, it seemed to me that the custom test storage class `PipelineNoPathStorage` was the cause of the error.